Packs/PAN-OS/Integrations/Panorama/Panorama.py

```diff
@@ -5411,186 +5444,1331 @@ def panorama_show_location_ip_command():
+    """
+    pretty_rule = {
+        'Name': rule.get('@name')
+    }
+    if 'application' in rule and 'member' in rule['application']:
+        pretty_rule['Application'] = rule['application']['member']
+    if 'file-type' in rule and 'member' in rule['file-type']:
+        pretty_rule['File-type'] = rule['file-type']['member']
+    if 'direction' in rule:
+        pretty_rule['Direction'] = rule['direction']
+    if 'alert-threshold' in rule:
+        pretty_rule['Alert-threshold'] = rule['alert-threshold']
+    if 'block-threshold' in rule:
+        pretty_rule['Block-threshold'] = rule['block-threshold']
+    if 'data-object' in rule:
+        pretty_rule['Data-object'] = rule['data-object']
+    if 'log-severity' in rule:
+        pretty_rule['Log-severity'] = rule['log-severity']
+    if 'description' in rule:
+        pretty_rule['Description'] = rule['description']
 
-        elif demisto.command() == 'panorama-get-address':
-            panorama_get_address_command()
+    return pretty_rule
 
-        elif demisto.command() == 'panorama-create-address':
-            panorama_create_address_command()
 
-        elif demisto.command() == 'panorama-delete-address':
-            panorama_delete_address_command()
+def prettify_data_filtering_rules(rules: Dict) -> Union[List[Dict], Dict]:
+    """
 
-        # Address groups commands
-        elif demisto.command() == 'panorama-list-address-groups':
-            panorama_list_address_groups_command()
+    Args:
+        rules: All the rules to prettify
 
-        elif demisto.command() == 'panorama-get-address-group':
-            panorama_get_address_group_command()
+    Returns: A list of all the rules compatible with our standards.
 
-        elif demisto.command() == 'panorama-create-address-group':
-            panorama_create_address_group_command()
+    """
+    if not isinstance(rules, list):
+        return prettify_data_filtering_rule(rules)
+    return [prettify_data_filtering_rule(rule) for rule in rules]
 
-        elif demisto.command() == 'panorama-delete-address-group':
-            panorama_delete_address_group_command()
 
-        elif demisto.command() == 'panorama-edit-address-group':
-            panorama_edit_address_group_command()
+def get_security_profile(xpath: str):
+    params = {
+        'action': 'get',
+        'type': 'config',
+        'xpath': xpath,
+        'key': API_KEY
+    }
 
-        # Services commands
-        elif demisto.command() == 'panorama-list-services':
-            panorama_list_services_command()
+    result = http_request(URL, 'GET', params=params)
 
-        elif demisto.command() == 'panorama-get-service':
-            panorama_get_service_command()
+    return result
 
-        elif demisto.command() == 'panorama-create-service':
-            panorama_create_service_command()
 
-        elif demisto.command() == 'panorama-delete-service':
-            panorama_delete_service_command()
+def get_security_profiles_command():
```

Comment:
Please consider add a `prettifier` function. this function is a bit hard to track